pkgs/top-level/aliases.nix

```
@@ -194,7 +194,15 @@ mapAliases ({
   gdb-multitarget = gdb; # added 2017-11-13
   gdk_pixbuf = gdk-pixbuf; # added 2019-05-22
   gettextWithExpat = gettext; # 2016-02-19
-  git-hub = gitAndTools.git-hub; # added 2016-04-29
+  gitAndTools = self // { # added 2021-01-14
```

Should we also warn here that `gitAndTools` is deprecated, so people can begin migrating away?

```suggestion
  gitAndTools = builtins.trace "gitAndTools has been deprecated and is now an alias to the top level" (self // { # added 2021-01-14
```

or something.

---

I thought about that, but no other aliases have warnings, and I’m not sure it really matters if people are still using gitAndTools.  I was also worried about interactions with imperative package management — would those users have to reinstall all their packages?

But I’m happy to reconsider.  What do you think?

---

No, you're right. It being an alias means it's not possible to use it inside Nixpkgs, but what does it really matter if people are using it in their own personal configs?